application.js

```diff
@@ -8,46 +8,53 @@
 function runChallenges() {
 
   // Ex 1. Read the content of the email input
-  var email = null;  // TODO: replace null, keep the email variable.
+  var email = $('#email').val();;  // TODO: replace null, keep the email variable.
 
 
   // Ex 2. Fill the content of the email input with your email
-
+  email = $('#email').val('toine.thompson@gmail.com');
 
 
   // Ex 3. Replace the email hint (next to the input) with 'This is my email now'
   //       The text should be emphasized using a <strong> tag
-
+  var emailHint = $('#email-hint');
+  var text = $('<strong>').text("This is my email now");
+  text.appendTo(emailHint);
 
   // Ex 4. Add the .blue CSS class to the table header cells (th elements)
-
+  $('th').addClass('blue');
 
 
   // Ex 5. Count the number of table body rows there are (team count!)
-  var team_count = 0;  // TODO: replace 0, keep the team_count variable.
+  var team_count = $('tbody > tr').length;  // TODO: replace 0, keep the team_count variable.
 
 
 
   // Ex 6. Say there is a 15th team added to the table.
   //       Add a row at the bottom, this new team should have zero points.
-
+  var row = $('<tr>').html('<td>' + (team_count + 1) + '</td><td>New team</td><td>0</td></tr>');
+  $('tbody').append(row);
 
 
   // Ex 7. Write some code to sum all points given to all teams
   var sum = 0;  // TODO: replace 0 with your sum result, keep the sum variable.
 
-
+   var liste = $('tbody > tr');
+   for (var i = liste.length - 1; i >= 0; i--) {
+    var number = parseInt(liste[i].lastChild.innerHTML);
+    sum = sum + number;
+   }
```

Comment:
A loop is not necessery here


    var row = $('<tr>').html('<td>' + (team_count + 1) + '</td><td>New team</td><td>0</td></tr>');
    $('tbody').append(row);